packages/ui/src/components/molecules/Carousel/styles.scss

```
@@ -4,33 +4,38 @@
   // --------------------------------------------------------
 
   // Track
-  --fs-carousel-padding-desktop           : var(--fs-spacing-0) 6.25rem;
+  --fs-carousel-padding-mobile            : 0 var(--fs-grid-padding);
+  --fs-carousel-padding-desktop           : 0 calc((100vw - 1230px)/2);
 
   // Item
-  --fs-carousel-item-width-mobile         : 80%;
-  --fs-carousel-item-width-desktop        : 15rem;
-  --fs-carousel-item-margin-right         : var(--fs-spacing-0);
+  --fs-carousel-item-width-mobile         : 60%;
+  --fs-carousel-item-width-tablet         : 14.2rem;
+  --fs-carousel-item-width-desktop        : 14.4rem;
+  --fs-carousel-item-margin-right         : var(--fs-spacing-3);
 
   // Controls
   --fs-carousel-controls-width            : 3.125rem;
   --fs-carousel-controls-height           : var(--fs-carousel-controls-width);
-  --fs-carousel-controls-position-top     : calc((100% - var(--fs-spacing-7) - var(--fs-spacing-1)) / 2);
   --fs-carousel-controls-bkg-color        : var(--fs-color-neutral-0);
   --fs-carousel-controls-border-radius    : var(--fs-border-radius-circle);
   --fs-carousel-controls-box-shadow       : var(--fs-shadow-darker);
   --fs-carousel-controls-icon-color       : var(--fs-color-neutral-7);
 
-  --fs-carousel-controls-control-left     : var(--fs-spacing-5);
+  --fs-carousel-controls-control-left     : var(--fs-spacing-4);
   --fs-carousel-controls-control-right    : var(--fs-carousel-controls-control-left);
+  --fs-carousel-controls-control-max-left : calc((var(--fs-grid-max-width)/4) - var(--fs-spacing-11));
```

Maybe you'll need to fix the token list columns after this 🤡 

```suggestion
  --fs-carousel-controls-control-max-left : calc(-1 * var(--fs-spacing-11));
  --fs-carousel-controls-control-max-right: var(--fs-carousel-controls-control-max-left);
```